### PR TITLE
Improve the example URL in the readme

### PR DESCRIPTION
**Pull request by tas50**
The old URL was confusing and a troll

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ default['supermarket_omnibus']['package_version'] = '1.2.3'
 default['supermarket_omnibus']['package_repo'] = 'current'
 
 # OR, specify a Supermarket package explicitly from a location of your choosing
-default['supermarket_omnibus']['package_url'] = 'http://bit.ly/98K8eH'
```
Comment (robbkidd) [inline README.md]: funny ... but no

Comment (irvingpop) [inline README.md]: That was a long con!
hats off to you

**Comment by robbkidd** [comment]
@tas50 Your README update broke Chef 13. Could you fix it?

tenor-95985443

**Comment by tas50** [comment]
It's a chef bug actually. I brought it up in chef-dev, but there's no way to fix that right now

**Comment by robbkidd** [comment]
Because it's only a README change, mergin' it dirty.

File: README.md (hunk 1 above)
Comment (irvingpop) [inline README.md]: I shit you not: https://twitter.com/rickastley/status/890617797956456448

Comment (robbkidd) [inline README.md]: solid

**Pull request merged**